composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "league/monga",
+    "name": "devitek/monga",
```

Comment:
The vendor should stay as "league"

Reply:
Oops, my bad, I published a new version on my fork, I forget that the PR will update too. I'll fix it right now !